src/ontology/uberon.Makefile

```diff
@@ -124,11 +124,15 @@ $(OWLSRC): $(SRC) $(COMPONENTSDIR)/disjoint_union_over.ofn $(REPORTDIR)/$(SRC)-g
 # For the temporary injection of property chains, see
 # <https://github.com/obophenotype/uberon/issues/2381>
 DEVELOPS_FROM_CHAIN=$(COMPONENTSDIR)/develops-from-chains.owl
+ifeq ($(GH_ACTION),true)
+MATERIALIZE=
+else
+MATERIALIZE = materialize -T $(CONFIGDIR)/basic_properties.txt -r elk
+endif
```

Comment:
Could we add a small comment to give the rationale for this? No need for any extensive explanation, but something like:

```
# The 'materialize' step is too resource-intensive for the GitHub Actions runners.
# See <https://github.com/obophenotype/uberon/pull/3087/>
```

Reply:
Oups, too late! ^^'

Reply:
just do it after the fact, i will approve, no need to run qc